portfolio/src/main/webapp/script.js

```diff
@@ -88,10 +92,7 @@ function fetchComments() {
   });
 }
 
-window.onload = (event) => {
-  fetchComments();
-  changeCommentFormDisplay();
-};
+document.onload = fetchComments();
```

Comment:
In case you were curious on the difference window.onload vs document.onload, see: 
https://stackoverflow.com/questions/588040/window-onload-vs-document-onload

I believe both would work in this instance. Though the difference is good to know in case you run into a bug due to the difference between the two.